module/os/linux/zfs/zfs_vfsops.c

```diff
@@ -1717,7 +1717,7 @@ zfs_vget(struct super_block *sb, struct inode **ipp, fid_t *fidp)
 			 * Must have an existing ref, so igrab()
 			 * cannot return NULL
 			 */
-			VERIFY3P(igrab(*ipp), !=, NULL);
+			zhold(ITOZ(*ipp));
```

Comment:
Let's leave this as `igrab()`.  It's in the Linux specific code already, and switching it means it does a pointless `ITOZ(ZTOI())`.  Plus the comment above references `igrab()`.

Reply:
I'd rather modify the comment if it works for you, then a grep for igrab on the codebase should return the `zhold` macro and `zfs_zget`; this use here is exactly what `zhold` translates to, so I think it would make sense to convert it - and only keep that one igrab in zget as is, because there we actually need the result of the condition.